cfme/tests/automate/custom_button/test_rest_custom_button.py

```diff
@@ -159,3 +180,41 @@ def test_edit(self, buttons_groups, appliance, from_detail):
             )
             condition.reload()
             assert condition.description == edited[index].description == record[0].description
+
+
+@pytest.mark.meta(blockers=[BZ(1745198)], automates=[1737449])
+def test_associate_unassigned_buttons_rest(appliance, group, buttons):
+    """Test associate unassigned button with group
+
+    Bugzilla:
+        1737449
+        1745198
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/4h
+        caseimportance: medium
+        startsin: 5.10
+        casecomponent: CustomButton
+        tags: custom_button
+    """
+
+    # Add associate unassigned buttons to group with rest
+    set_data = group.set_data
+    set_data["button_order"] = [int(b.id) for b in buttons]
+    data = {"set_data": set_data}
+    group.action.edit(**data)
+    assert_response(appliance)
+
+    # Check association with UI
+    gp = appliance.collections.button_groups.instantiate_with_rest(group.id)
+
+    # Point to new object type so that teardown button(removed with rest) and new buttons, group
+    # (created with rest) reflect on UI without any selenium exception.
+    view = navigate_to(gp, "ObjectType")
+    view.browser.refresh()
```

Comment:
Why refresh here?

Reply:
@mshriver we are deleting and creating button with `rests` as per test steps... Its parametrized test. For first test it will work as navigation will start from beginning.  but, when we are on same page and other test (browser session) start run to display buttons (creation of new buttons and old test teardown means deletion) on page; we need browser refresh.

Reply:
There is a `force` kwarg for `navigate_to` that should force navigation to the page, and won't allow is_displayed to keep it from navigating from the start of the 'stack' of navigation for that page.  I think it would be better to use it here, but at the very least an in-line comment explaining its need.